src/Speckle.Objects/Geometry/Surface.cs

```diff
@@ -11,45 +12,43 @@ namespace Speckle.Objects.Geometry;
 [SpeckleType("Objects.Geometry.Surface")]
 public class Surface : Base, IHasBoundingBox, IHasArea, ITransformable<Surface>
 {
-  /// <summary>
-  /// Constructs a new empty <see cref="Surface"/>
-  /// </summary>
+  [Obsolete("Constructor should only be used by serializer, use one of the other constructors instead")]
   public Surface()
   {
-    applicationId = null;
-    pointData = new List<double>();
+    pointData = [];
   }
 
-  /// <summary>
-  /// Constructs a new empty <see cref="Surface"/>
-  /// </summary>
-  /// <param name="units">The units this surface is modeled in</param>
-  /// <param name="applicationId">This surface's unique identifier on a specific application</param>
-  public Surface(string units = Units.Meters, string? applicationId = null)
+  public Surface(List<List<ControlPoint>> controlPoints)
```

Comment:
Do we need this constructor? I know the rhino surface converter is currently using it, but isn't it better to pass the control points into pointData instead (and not even convert points in the rhino surface converter)?

Reply:
Just had a chat with Claire, tldr: this is needed because a surface can be set from either control points OR encoded (flat) control point data.
There's just no nice way to do that with the `requried` keyword, so constructors are required.

Yes it means `surface` is a bit of an ugly ducking, and if we wanted to redesign the class, I'm confident we could create something much nicer. However, since this class is no longer since the introduction if XBrep, I didn't want to go too far refactoring this.
This was just an easy win.